trust-quorum/src/task.rs

```diff
@@ -230,18 +454,77 @@ impl NodeTask {
         }
     }
 
+    // TODO: Process `ctx`: save persistent state
```

Comment:
What's ctx here?

Reply:
This is the internal `NodeCtx` of a trust-quorum node. It's a structure that contains the `PerisistentState`. The next PR in the series removes this comment and saves the `PersistentState` to a ledger.